### PR TITLE
Request list view

### PR DESCRIPTION
**Pull request by ath0111**
I created list view of requests.

Screen Shot 2020-04-02 at 12 19 39 PM

### CODE REVIEW

File: src/components/HomeView.tsx [hunk 1]
```diff
@@ -6,11 +6,11 @@ import User from '../models/User';
...
-  useEffect(() => {
-    if (user && user.maker) {
-      history.push('/work');
-    }
-  }, []);
+  // useEffect(() => {
+  //   if (user && user.maker) {
+  //     history.push('/work');
+  //   }
+  // }, []);
```
Comment (ChristianAlexa) [inline src/components/HomeView.tsx:13]: Should we just removing this instead of commenting out?

File: src/components/MakerView.tsx [hunk 2]
```diff
@@ -64,6 +64,8 @@ export default function MakerView() {
     getMakers();
   }, []);
 
+  console.log(makers)
+
```
Comment (ChristianAlexa) [inline src/components/MakerView.tsx:68]: can we remove console.log

File: src/components/WorkView.tsx [hunk 3]
```diff
@@ -83,6 +75,7 @@ const WorkView: React.FC<{ user: User }> = ({ user }) => {
 
                 <tbody>
                   {work.map((w, index) => {
+                    console.log(w)
```
Comment (ChristianAlexa) [inline src/components/WorkView.tsx:78]: remove console.log

File: src/components/RequestView.js [hunk 4]
```diff
@@ -14,13 +14,15 @@ class RequestView extends Component {
 
   componentDidMount() {
     axios.get(buildEndpointUrl('requests')).then((res) => {
+      console.log(res);
```
Comment (ChristianAlexa) [inline src/components/RequestView.js:17]: remove console.log

File: src/components/RequestView.js [hunk 5]
```diff
@@ -14,13 +14,15 @@ class RequestView extends Component {
...
       this.setState({
         requests: res.data
       });
     });
   }
 
   render() {
+    console.log(this.state);
```
Comment (ChristianAlexa) [inline src/components/RequestView.js:25]: remove console.log

**Comment by ath0111** [comment]
We need the code that I commented out. I fixed it.
And removed all 'console.log's.
thanks

**Comment by ChristianAlexa** [comment]
👍 approved

**Pull request merged**